azure/packages/azure-service-utils/README.md

```diff
@@ -2,7 +2,11 @@
 
 A set of helper utilities for building backend APIs for use with Azure Fluid Relay service.
 
-<!-- AUTO-GENERATED-CONTENT:START (README_DEPENDENCY_GUIDELINES_SECTION:includeHeading=TRUE) -->
+## generateToken
```

Comment:
For reviewers: this section was hidden a bit between various auto-generated sections. Pulled it up to the top to enhance discoverability.